### ISSUE TITLE
Strips HTML classes anything; DIV, UL, etc.

### ISSUE THREAD

**Issue by JaySMa** [DNN-Connect/CKEditorProvider#129]
Have DNN v9.4.1 installed; see: https://github.com/dnnsoftware/Dnn.Platform/issues/3139

When i enter my source code and enter is accordingly it works fine.  E.g.
`<ul class="widget-links fa-ul"> .... </ul>`

But if I go back into the source code to edit data, I've found that the editor has remove the class and it just shows:
`<ul> ... </ul>`

Even tried doing a simple, blank Parallax DIV, but the edit just removes it like it doesn't exist.
`<div class="parallax parallaxGIS">&nbsp;</div>`

Why is the editor doing this?

**Comment by Timo-Breumelhof** [comment]
That's default CKEditor behaviour. You need to a add an extra attribute to the config to prevent this: https://stackoverflow.com/questions/15659390/ckeditor-automatically-strips-classes-from-div

**Comment by JaySMa** [comment]
Hello..
I first tried what is suggested in the stackoverflow but even with setting allowedcontent to true, but is still wouldn't work even after clearing the cache on the browser, restarting IIS, etc.

Then I remember that DNN had the HTML Editor Manger and surprisingly, this AllowedContent setting was still set to false.  For testing purposes, changed this to true and restarted and cleared everything.  It worked.

Ended up setting this back to false and set the ExtraAllowedContent to `*(*);*{*}`  which according to the StackOverflow posting, allows for any class or inline styling.  This worked as it should.

Thank you for your assistance.
